README.md

```diff
@@ -1,2 +1,48 @@
-# medium-guidelines
-Guidelines and good practices for Medium publications
+# Medium Guidelines
+
+This guidelines are made for people who love writing amazing stories on Medium. As developers, we often miss the opportunity to get benefits from sharing our experiences, knowledge and thinking. 
```

Comment:
Please, correct with `These guidelines`. I also believe we should develop this introduction a bit more to be more explicit about the goal of this document.